Description: Fix cliver import
 Since require is now relative to the directory which it is in, the
 File.expand_path is not necessary.

Author: Andrew Lee (李健秋) <andrew.lee@collabora.co.uk> 
Last-updated: 2015-08-28
Forwarded: not-needed
Bug: not-needed

--- a/lib/cliver.rb
+++ b/lib/cliver.rb
@@ -1,6 +1,6 @@
 # encoding: utf-8
 
-require File.expand_path('../core_ext/file', __FILE__)
+require "core_ext/file"
 
 require 'cliver/version'
 require 'cliver/dependency'
